README.md

```diff
@@ -29,12 +29,16 @@ Create your database and run migrations
 php bin/console doctrine:database:create
 php bin/console doctrine:migrations:migrate
 ```
+Run the database seeder if you want to generate dummy data.
+```
+php bin/console doctrine:fixtures:load
+```
 
 That's it, you are ready
 
 ### Develop
 
-If you use the [Symfony CLI](https://symfony.com/download) you can launch a development server with `symfony server:start`, otherwise you need to configure your Apache installation to serve the files from the `public` folder.  
+If you use the [Symfony CLI](https://symfony.com/download) you can launch a development server with `symfony server:start`, otherwise you need to configure your Apache installation to serve the files from the `public` folder.
```

Comment:
The whitespace at the end of the lines shouldn't be removed

(this is due to the editorconfig file, not your fault)

Reply:
Fixed